libs/core-ui/src/lib/Cohort/ManualCohortManagement/CohortEditorFilter.tsx

```diff
@@ -42,47 +51,83 @@ export interface ICohortEditorFilterProps {
   setComparison(ev?: React.FormEvent<IComboBox>, item?: IComboBoxOption): void;
   setNumericValue(
     delta: number,
-    column: IJointMeta,
     index: number,
-    stringVal: string
+    stringVal: string,
+    range?: INumericRange
   ): string | void;
   saveState(index?: number): void;
   cancelFilter(): void;
 }
+const maxLength = 18;
 export class CohortEditorFilter extends React.Component<ICohortEditorFilterProps> {
-  private readonly dataArray: IComboBoxOption[] = new Array(
-    this.props.jointDataset.datasetFeatureCount
-  )
-    .fill(0)
-    .map((_, index) => {
-      const key = JointDataset.DataLabelRoot + index.toString();
-      return {
-        key,
-        text: this.props.jointDataset.metaDict[key].abbridgedLabel
-      };
-    });
+  private readonly dataArray: IComboBoxOption[] = this.props.isRefactorFlightOn
```

Comment:
I thought we agreed we won't have a flight for these changes but just either have both cohorts side by side or an adapter to convert between them?